src/form.js

```diff
@@ -29,8 +31,11 @@ function setupForm() {
   submitButton.addEventListener('click', () => {
     const todoFormData = runSubmit();
     if (todoFormData) {
-      const todo = new TodoItem(todoFormData);
-      projects.currentProject.addTodo(todo);
+      const todoData = unpackTodoFormData(todoFormData);
+      const todoObject = new TodoItem(todoData);
+      projects.currentProject.addTodo(todoObject);
+      refreshTodoListDisplay(projects.currentProject);
+      new ProjectStorage().save(projects.list);
```

Comment:
For every click event you will create a new project storage. It might be best to create it in one place and use that instance instead.